include/groonga/util.h

```diff
@@ -52,6 +52,48 @@ grn_inspect_indent(grn_ctx *ctx,
                    grn_obj *buffer,
                    grn_obj *content,
                    const char *indent);
+/**
+ * \brief Inspect specified object and produce an indented textual
+ *        representation.
+ *
+ * \since 4.0.8
+ *
+ * This function combines \ref grn_inspect and \ref grn_inspect_indent. It first
+ * inspects the given \p obj into a temporary buffer, then prepends the given
+ * \p indent string to each line of that inspected text and stores the result in
+ * \p buffer.
+ *
+ * \note The \p indent string is only applied when the inspected text contains
+ *       one or more newline characters (when the output spans multiple lines).
+ *
+ * For example usage:
+ * ```c
+ * grn_obj inspected;
+ * GRN_TEXT_INIT(&inspected, 0);
+ * grn_inspect_indented(ctx, &inspected, obj, "***");
+ * printf("%.*s\n", (int)GRN_TEXT_LEN(&inspected), GRN_TEXT_VALUE(&inspected));
+ * GRN_OBJ_FIN(ctx, &inspected);
+ * ```
+ *
+ * If \p obj is \ref GRN_TABLE_PAT_KEY table, it prints like the following.
+ * ```
+ * ***#<table:pat Users key:ShortText value:(nil) size:7 columns:[] \
+ *      default_tokenizer:(nil) normalizer:(nil) \
+ *      keys:["a", "b", "c", "d", "e", "f", "g"] subrec:none nodes:{
+ * ***4{0,5,0}
+ * ***  L:2{0,6,0}
+ * ***    L:1{0,7,0}
+ * ***      L:0{0,0,0}
+ * ***      R:1{0,7,0}("a")[01100001]
+ * ```
+ *
+ * \param ctx     The context object.
+ * \param buffer  The buffer where the indented inspected text will be stored.
+ * \param obj     The target object to inspect.
+ * \param indent  The indentation string to prepend to each line of output.
```

Comment:
```suggestion
 * \param ctx The context object.
 * \param buffer The buffer where the indented inspected text will be stored.
 * \param obj The target object to inspect.
 * \param indent The indentation string to prepend to each line of output.
```

これまでのドキュメントだと、揃えてなかったのでそっちに合わせますか。

Reply:
fix: [8e2263c](https://github.com/groonga/groonga/pull/2340/commits/8e2263c510bf725b090f7923be27f0a0d77b4736) Sure!